Modules/_operator.c

```diff
@@ -1696,6 +1745,12 @@ static PyMethodDef methodcaller_methods[] = {
      reduce_doc},
     {NULL}
 };
+
+static PyMemberDef methodcaller_members[] = {
```

Comment:
Why is this being added?

Reply:
It seems the logic has been introduced by #20026 instead of introducing a type slot for tp_vectorcall_offset. [(Documentation is here)](https://docs.python.org/3.11/c-api/structures.html?highlight=__vectorcalloffset__#pymemberdef-offsets)